sources/reference/admiral-vault.md

```diff
@@ -34,3 +34,26 @@ Now that initialization is complete, you can view the logs of the installation,
 <img src="../../images/reference/admiral/admiral-vault-logs.png" alt="Vault logs">
 
 <img src="../../images/reference/admiral/admiral-vault-config.png" alt="Vault configuration">
+
+
+## Unsealing Vault
+
+If the server reboots or vault goes in a [sealed](https://www.vaultproject.io/docs/concepts/seal.html) state, it will be required to
+`unseal` it manually (for now). The following steps unseal the vault server.
+
+- ssh into the machine running vault server
+- run following commands
```

Comment:
`Run the following commands:`

I would suggest making both of these bullet points complete sentences.  And add "the" before "following."